### ISSUE TITLE
during build_application of deploy, unknown error occured

### ISSUE THREAD

**Issue by manyone** [stackblitz/bolt.new#590]
### Describe the bug

deploy fails at build application step  (unknown reason)

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-mwresr

### Steps to reproduce

click deploy

### Expected behavior

app is created in netlify

### Screen Recording / Screenshot

![image](https://github.com/user-attachments/assets/12bf8936-2291-4d37-8480-34da51119c6b)
unknon w

### Platform

```
Browser name = Firefox
Full version = 131.0
Major version = 131
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:131.0) Gecko/20100101 Firefox/131.0
performance.memory = undefined
Username = manyone
Chat ID = e8b0819dd6f5
```

### Additional context

Error:
```
An unknown error occured while running an action.
```

**Comment by manyone** [comment]
i found out what caused it - i made sure to note what was in the terminal window at that last time it happened and here it is: 

vite:terser not found...new...dependency... need to install
trace at leadtercerFath, at renderChunk...

Note: i typed this back to bolt.new and it fixed it.  my deploy worked.  fixed by bolt.new

**Comment by kc0tlh** [comment]
Thanks for the report and patience during this beta period! Tracking all deployment related bug fixes in issue #499
